daq/topology.py

```diff
@@ -417,24 +421,33 @@ def _add_dhcp_reflectors(self, acl_list):
         self._add_acl_rule(acl_list, dl_type='0x800', allow=False,
                            nw_proto=17, udp_src=67, udp_dst=68)
 
+    def _add_dot1x_incoming_rule(self, incoming_acl, secondary_acl):
+        for devices in self._set_devices.values():
+            for device in devices:
+                if device and device.gateway:
+                    vlan = self._get_port_vlan(device.port.port_no)
+                    test_ports = device.gateway.get_possible_test_ports()
+                    if test_ports:
+                        self._add_dot1x_reflector(incoming_acl, vlan, test_ports, True)
+                    device_port = device.port.port_no
+                    if device_port:
+                        self._add_dot1x_reflector(secondary_acl, vlan, [device_port], True)
+
+    def _add_dot1x_reflector(self, acl, vlan, ports, is_trunk):
```

Comment:
generally best to avoid booleans and use named arguments, since it's more descriptive and future-proof, e.g:

`def _add_dot1x_reflector(self, acl, ports, vlan_vid=None, out_vlan=None)`

which can then translate to call like:

`self._add_dot1x_reflector(portset_acls[port_set], [self.PRI_TRUNK_PORT], vlan_vid=vlan_vid)`

It's also not a _reflector_ (quite different than what DHCP is doing), it's a allow rule, or shortcut, or something along those lines.